src/garrysmod/gamemodes/zombiesurvival/gamemode/zombieclasses/agiledead.lua

```diff
@@ -10,6 +10,7 @@ CLASS.BetterVersion = "Fast Zombie"
 CLASS.SWEP = "weapon_zs_agiledead"
 
 CLASS.Unlocked = true
+CLASS.Hidden = true
 
 CLASS.Health = 125
 CLASS.Points = CLASS.Health/GM.NoHeadboxZombiePointRatio
```

Comment:
I spoke with @johnnymoe. Doesn't seem like this is something that'd ever come back. I'd consider removing it all together, after first checking to make sure there aren't any other instances of it across the rest of the codebase.

Reply:
Will be keeping this class for compatibility reasons. If we decide in future to change direction regarding compatibility we can remove it then.